http3/roundtrip.go

```diff
@@ -17,6 +17,30 @@ import (
 	"github.com/quic-go/quic-go"
 )
 
+// Settings are HTTP/3 settings that apply to the underlying connection.
+type Settings struct {
+	// Support for HTTP/3 datagrams (RFC 9297)
+	EnableDatagram bool
+	// Extended CONNECT, RFC 9220
+	EnableExtendedConnect bool
+	// Other settings, defined by the application
```

Comment:
What do you mean? It's two different RFCs, so there's two different options.

Reply:
I mean why isn't settings typed as
```
Settings map[uint64]uint64
```

`EnableDatagram` / `EnableExtendedConnect` could also be part of the `Other` map

Reply:
The `Other` is pretty much a hack, so that WebTransport can add its setting there. I'd much rather just have bools, and let the http3 package deal with serialization.